Add dockerfile

menzow opened the pull request:
Added a Dockerfile so users can build slurp without having go installed on their system. The final image size is just 13.5 MB.

I pushed the final result to hub.docker.com here:
https://hub.docker.com/r/menzo/slurp/

You can build it yourself using the following steps:
```
$ git clone https://github.com/menzow/slurp slurp-menzow
$ cd slurp-menzow
$ docker build -t slurp .
$ docker build -t slurp:large -f large.Dockerfile .
```

You will now have two images. One for regular scans, and one tagged ':large` for scans using the large permutation set. To use them run one of the following commands:

```
$ docker run --rm -ti slurp domain -t example.com
$ docker run --rm -ti slurp:large domain -t example.com
```

To run slurp using docker without building the image yourself, you can run:
```
$ docker run --rm -ti menzo/slurp domain -t example.com
$ docker run --rm -ti menzo/slurp:large domain -t example.com
```

bbb31 commented on Dockerfile:
```diff
@@ -0,0 +1,33 @@
+COPY *.json ./
+
+# Run slurp when the container starts
+ENTRYPOINT [ "/bin/slurp" ]
```
Missing a new line here.

bbb31 commented on large.Dockerfile:
```diff
@@ -0,0 +1,2 @@
+FROM slurp
+RUN mv permutations-large.json permutations.json
```
Missing a new line here.

menzow commented:
Thanks for the feedback @bbb31.

I added the empty line to the Dockerfile. Additionally I removed `large.Dockerfile` in favor of selecting the permutations through a build argument. This is cleaner, reduces the image size and allows for more flexibility.

Example usage:
```
$ docker build -t slurp .
$ docker build -t slurp:large --build-arg 'permutations=permutations-large.json' .
```
